build/build.sh

```diff
@@ -79,6 +79,12 @@ if [ "$FULL_BUILD" == "true" ]; then
 
   yarn build
 
+  # Workaround: TeamCity expects the dmg to be in dist/mac, but in the new electron-builder
+  # it's put directly in dist/ (the right way to solve this is to update the TeamCity config)
+  if $OSX; then
+    cp dist/*.dmg dist/mac
```

Comment:
Wouldn't be better to change TeamCity config directly?

Reply:
It would, but I figured we would worry about it later. It's already going to be a little complicated to roll out auto update.